### ISSUE TITLE
Question to conf true\false is case insensitive?

### ISSUE THREAD

**Issue by Bxaa** [doitsujin/dxvk#2270]
Is true equal True while conf parsing?

**Comment by DadSchoorse** [comment]
True and False are case sensitive. 
https://github.com/doitsujin/dxvk/blob/master/src/util/config/config.cpp#L576

**Comment by doitsujin** [comment]
The example file even kind of tells you that `True` is the one to use.

Also, `tolower` is locale sensitive, i.e. **really** bad.

**Comment by Bxaa** [comment]
.ToLower(CultureInfo.InvariantCulture)

**Comment by K0bin** [comment]
We'll keep that in mind when we rewrite DXVK in C#...

**Comment by Bxaa** [comment]
setlocale

**Comment by doitsujin** [comment]
I don't really see how this is a big deal in the first place, it hasn't been for the past three years and given your behaviour in this thread i'm honestly inclined to leave it as-is out of spite.

**Comment by Bxaa** [comment]
Don't get me wrong - I wasted a lot of time because of this misunderstanding. I just couldn't imagine that it could be so bad.
English is not my native language. I didn't mean to offend anyone.

I am not a C++ programmer, and I hardly use it, but even for me there is no difficulty in this.
If you think the current implementation of config parsing is good, then let it stay as it is.
I don't think so, but that's my personal opinion. I would be fired from my job if I made such an implementation :)
